README.md

```diff
@@ -39,6 +39,7 @@ Execute the development environment:
 pnpm dev
 ```
 
+Test
```

Comment:
Test succeeded?

Reply:
:( sorry - that was because I wasn't sure if it works when I changed the github user on my terminal